src/main/java/lotto/LottoWin.java

```diff
@@ -22,4 +25,22 @@ public int matchNum() {
     public int winPrice() {
         return this.winPrice;
     }
+
+    public static LottoWin valueOf(int matchNum, boolean matchBonus) {
+        return Arrays.stream(LottoWin.values())
```

Comment:
```suggestion
        return Arrays.stream(LottoWin.values())
                .filter(v -> lottoWinBonusFilter(v, matchNum, matchBonus))
                .findFirst()
                .orElse(LAST_PLACE);
```

생략 가능한 문법은 제거하고 한 줄에 한 기능씩 작성해주시면 읽기 좋은 코드가 됩니다.